php/async/Exchange.php

```diff
@@ -102,10 +102,11 @@ public function fetch($url, $method = 'GET', $headers = null, $body = null) {
                 $request_browser_options = array( 'tcp' => $proxy, 'dns' => false );
                 $this->set_request_browser($request_browser_options);
             } else if ($socksProxy !== null) {
-                if (!class_exists('\Clue\React\Socks\Client')) {
+                $className = '\\Clue\\React\\Socks\\Client';
+                if (!class_exists($className)) {
                     throw new NotSupported($this->id . ' - to use SOCKS proxy with ccxt, at first you need install module "composer require clue/socks-react"');
                 }
-                $proxy = new \Clue\React\Socks\Client($socksProxy);
+                $proxy = new $className($socksProxy);
```

Comment:
@ttodua why this change?

Reply:
as top-post says:
> edited dynamic class instantiation (because, without it, linter warned: https://imgsh.net/a/dPAsmoE.png )

Reply:
@ttodua have you tested it? sometimes these updates are tricky

Reply:
yes, responsibly say that I have tested in debugger and it worked same as it was before. you can see here (click "execute") to see those approaches in php: https://onlinephp.io/c/6e5d2